src/compile/common.ts

```diff
@@ -18,17 +18,18 @@ import {RepeaterValue, RepeatModel} from './repeat';
 import {UnitModel} from './unit';
 
 
-export function buildModel(spec: Spec, parent: Model, parentGivenName: string, repeater: RepeaterValue, config: Config): Model {
+export function buildModel(spec: Spec, parent: Model, parentGivenName: string,
```

Comment:
Well, for users this would be invisible. it's just less code and in the unit you can directly read from the config.